bin/sugar-serial-number-helper

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+device=/sys/class/dmi/id
+
+read value < $device/product_serial
+echo "$value"
```

Comment:
Using ```cat``` would add delay while another binary is started as a subprocess of the shell.  ```read``` is a built-in for the shell.  The device variable follows the style of ```sugar-backlight-helper```.